R/app-server.R

```diff
@@ -8,7 +8,9 @@ app_server <- function(input, output, session) {
     showModal(
       modalDialog(
         title = "Not logged in",
+        # nolint start
```

Comment:
Only a question - why use the `#nolint start/end` for a single line? I noted in other places you have a long line and did `#nolint` at the end. I see other places that use the `start/end` method for blocks of code, which makes sense.

Reply:
Because I made a mistake 😅 . I meant to use `# nolint` for single lines, and a block for anything that was more than one line.